curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9eb4a5d68f12f32d9b440.md

```diff
@@ -9,6 +9,15 @@ dashedName: step-89
 
 Remember that `document.querySelectorAll` returns a `NodeList`, which is array-like but is not an array. However, the `Array` object has a `.from()` method that accepts an array-like and returns an array. This is helpful when you want access to more robust array methods, which you will learn about in a future project.
 
+```js
+const str = 'Hello';
+const chars = Array.from(str);
+
+console.log(chars); // Output: ['H', 'e', 'l', 'l', 'o']
+
+const divArray = Array.from(document.querySelectorAll('div'));
```

Comment:
We shouldn't have two different examples like this.
It looks confusing because some will wonder what one has to do with the other.

I vote we just use the second example that was proposed.

Reply:
@alagumuthiah 

Use this instead 

https://github.com/freeCodeCamp/freeCodeCamp/pull/53490#discussion_r1474291517